dll/win32/shell32/shresdef.h

```diff
@@ -351,6 +351,15 @@
 #define IDC_RUNDLG_EDITPATH    12298
 #define IDC_RUNDLG_LABEL       12305
 
+/* Run As dialog */
+#define IDD_RUN_AS 23
+#define IDC_RUNAS_THIS   100
```

Comment:
Please use control IDs > 1000 since the range in the hundreds is reserved for strings.

Reply:
Eh, OK I guess but why does it matter? In which case would them overlapping cause an issue?